utils/cooperations/coopsAggregateOptions.js

```diff
@@ -0,0 +1,19 @@
+const getRegex = require('../getRegex')
+
+const coopsAggregateOptions = (params = {}, query) => {
+  const { skip = 0, limit = 5, status = 'all', order = 'updatedAt', orderBy = 'asc' } = query
+
+  const match = {}
+  const sort = {}
+
+  if (order === 'name') sort.initiatorFullName = orderBy
+  else if (order ==='subject') sort.subjectName = orderBy

```

Comment:
we do not need to sort by subject, only name and date

Reply:
okey, changed